src/MusicProcessing.jl

```diff
@@ -1,19 +1,14 @@
 module MusicProcessing
 
 using DSP
+using FFTW
 using FixedPointNumbers
 using Requires
-using SIUnits
-using SIUnits.ShortUnits
 using SampledSignals
 
 # types used for fixed-point 16-bit and 32-bit encoding
-typealias PCM16Sample Fixed{Int16, 15}
-typealias PCM32Sample Fixed{Int32, 31}
-
-# shortcut to SIQuantity type of frequency and time
-typealias Hertz SIUnits.SIQuantity{Int64,0,0,-1,0,0,0,0,0,0}
-typealias Seconds{R} SIUnits.SIQuantity{R,0,0,1,0,0,0,0,0,0}
+const PCM16Sample = Fixed{Int16, 15}
+const PCM32Sample = Fixed{Int32, 31}
 
 export Hz, kHz, s, ..
```

Comment:
`export Hz, kHz, s,` are still exported? What brings them into scope?

Reply:
Whoops I probably still need to remove them. But I believe that SampledSignals brings them into scope